### PR TITLE
<InputWithTags/> - fix focus style

### PR DESCRIPTION
**Pull request by ChenRoth**
### What changed
removed background-color when component has focus

### Why it changed

so the component would be aligned with all other WSR input components.

---

- [ ] Change is tested
- [X] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by hepiyellow** [comment]
The fact that this does not break any eyes test, means that it is not tested.

**Comment by ChenRoth** [comment]
either way, a test wouldn't fail now...

**Comment by hepiyellow** [comment]
thanks.
BTW: this resolves https://github.com/wix-private/wsr-issues/issues/237

**Comment by Faradey27** [comment]
@hepiyellow can we merge?

File: src/MultiSelect/MultiSelect.e2e.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ describe('MultiSelect', () => {
       .then(async () => {
         await browser.actions().sendKeys(protractor.Key.TAB).perform();
         await browser.actions().sendKeys(protractor.Key.TAB).perform();
+        await browser.actions().mouseMove(driver.element());
```
Comment (shlomitc) [inline src/MultiSelect/MultiSelect.e2e.js]: as this is now a different test - better have `eyes.checkWindow` before hovering, to make sure the previous focus styles are set.
do you think we should explain it also in the contribution guide?
https://github.com/wix/wix-style-react/blob/master/docs/contribution/TESTING.md#visual-testing

Comment (ChenRoth) [reply src/MultiSelect/MultiSelect.e2e.js]: @hepiyellow is this under your care now or do i need to do this?

**Pull request merged**